decentraland/unity-explorer
# fix: header origin is managed automatically

## What does this PR change?

Prevents ORIGIN header from being set and thus the following error:

`The header Origin is managed automatically, setting it may have no effect or result in unexpected behavior.`

## How to test the changes?

Test happy paths
Test mini-games
Ensure progress is saved (apart from that one bug we know about)

Ensure the bugs mentioned here still are fixed: https://github.com/decentraland/unity-explorer/pull/1388

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
